### ISSUE TITLE
[Header] Allow components in summary to be in front of buttons

### ISSUE THREAD

**Issue by rodolpheV** [KleeGroup/focus-components#1447]
# Current implementation
Buttons and Summary are in different stacking contexts. The impact is that no components in Summary can be in front of buttons, especially popins.

# Expected implementation
Put buttons and summary in the same stacking context. To do that, put them in a common div:
```
<HeaderScrolling>
       <div data-focus='sub-header'>
            <HeaderTopRow />
            <HeaderActions />
      </div>
      <HeaderContent />
</HeaderScrolling>
```
And set **this** common div as fixed with some z-index in `data-deployed=false` mode.

## Versions

Focus-core: 2.1.1
Focus-components: 2.1.7-1

**Comment by Hartorn** [comment]
@rodolpheV 
I agree, the whole header is messy for css stacks, position and so on.
Problem is to avoid regression at the same time.

@c3dr0x Could you give it a try ? There is also the thing with expanded or not.
